tools/torchci/clickhouse_query_perf.py

```diff
@@ -9,24 +10,45 @@
 import time
 from datetime import datetime, timedelta, timezone
 from functools import cache
-from typing import Optional
+from typing import Any, Dict, List, Optional
 
+from dotenv import load_dotenv
```

Comment:
did you add this for the clickhouse username/pw? do we want to add a .env.example then to the folder?

Reply:
added .env.example